MovieApp/AppViewController.swift

```diff
@@ -193,141 +276,115 @@ class AppViewController: UIViewController {
     
     
     private func createCollectionView() {
-        mainView.addSubview(overviewView)
+        scrollView.addSubview(overviewView)
         overviewView.backgroundColor = .white
         
-        overviewView.translatesAutoresizingMaskIntoConstraints = false
-        overviewView.topAnchor.constraint(equalTo: textOverImageView.bottomAnchor).isActive = true
-        overviewView.leadingAnchor.constraint(equalTo: mainView.leadingAnchor).isActive = true
-        overviewView.bottomAnchor.constraint(equalTo: mainView.bottomAnchor).isActive = true
-        overviewView.widthAnchor.constraint(equalTo: mainView.widthAnchor).isActive = true
-        
+        overviewView.snp.makeConstraints{ (make) in
+            make.leading.bottom.equalToSuperview()
+            make.top.equalTo(textOverImageView.snp.bottom)
+            make.width.equalToSuperview()
+        }
         
 
         let overview_label = UILabel()
         overviewView.addSubview(overview_label)
         overview_label.text = "Overview"
         overview_label.font = UIFont.boldSystemFont(ofSize: fontSizeBig)
         overview_label.textColor = .black
-        overview_label.translatesAutoresizingMaskIntoConstraints = false
-        overview_label.topAnchor.constraint(equalTo: overviewView.topAnchor, constant: bigSpace).isActive = true
-        overview_label.leadingAnchor.constraint(equalTo: overviewView.leadingAnchor, constant: padding).isActive = true
-        
+        overview_label.snp.makeConstraints{ (make) in
+            make.top.equalToSuperview().offset(bigSpace)
+            make.leading.equalToSuperview().offset(padding)
+        }
         
         
-        let overview_text = UITextView()
         overviewView.addSubview(overview_text)
-        overview_text.text = "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil."
+        
         overview_text.font = UIFont.systemFont(ofSize: fontSizeMedium)
         overview_text.textColor = .black
         overview_text.isEditable = false
         overview_text.isSelectable = false
-        overview_text.translatesAutoresizingMaskIntoConstraints = false
-        overview_text.topAnchor.constraint(equalTo: overview_label.bottomAnchor, constant: padding).isActive = true
-        overview_text.leadingAnchor.constraint(equalTo: overview_label.leadingAnchor).isActive = true
-        overview_text.trailingAnchor.constraint(equalTo: overviewView.trailingAnchor, constant: -padding).isActive = true
+        overview_text.snp.makeConstraints{ (make) in
+            make.top.equalTo(overview_label.snp.bottom).offset(padding)
+            make.leading.equalToSuperview().offset(padding)
+            make.trailing.equalToSuperview().inset(padding)
+        }
         overview_text.isScrollEnabled = false
         
```

Comment:
Also keeping these spacings always the same (one line, not two) makes code more readable.